test/pom.xml

```diff
@@ -40,6 +40,18 @@
             <version>${logback.classic.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+          <groupId>org.openjdk.jmh</groupId>
+          <artifactId>jmh-core</artifactId>
+          <version>1.21</version>
+          <scope>test</scope>
```

Comment:
Actually, it is not recommended to use jmh inside the unit tests. Unit tests affect jmh very much. It had better just use `public static void main(...)`